src/controllers/DrawingModeController.js

```diff
@@ -57,17 +58,23 @@ class DrawingModeController extends AnnotationModeController {
         }
     }
 
+    showButton(): void {
+        super.showButton();
+
+        this.buttonEl.title = this.intl.formatMessage({ ...messages.annotationDrawToggle });
```

Comment:
Do these objects need to be spread or does `this.intl.formatMessage(messages.annotationDrawToggle);` work?